octopus_deploy/README.md

```diff
@@ -0,0 +1,60 @@
+# Agent Check: Octopus Deploy
+
+## Overview
+
+This check monitors [Octopus Deploy][1] through the Datadog Agent. 
+
+Include a high level overview of what this integration does:
+- What does your product do (in 1-2 sentences)?
+- What value will customers get from this integration, and why is it valuable to them?
+- What specific data will your integration monitor, and what's the value of that data?
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The Octopus Deploy check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `octopus_deploy.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your octopus_deploy performance data. See the [sample octopus_deploy.d/conf.yaml][4] for all available configuration options.
+
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `octopus_deploy` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this integration.
+
+### Events
+
+The Octopus Deploy integration does not include any events.
+
+### Service Checks
+
+The Octopus Deploy integration does not include any service checks.
+
+See [service_checks.json][8] for a list of service checks provided by this integration.
```

Comment:
These appear to be at odds with each other. Which is correct?

Reply:
Removed one!